pkg/apis/application/v1alpha1/generated.proto

```diff
@@ -1010,7 +1010,9 @@ message KustomizeOptions {
 message ListGenerator {
   repeated k8s.io.apiextensions_apiserver.pkg.apis.apiextensions.v1.JSON elements = 1;
 
-  optional ApplicationSetTemplate template = 2;
+  optional string elementsYaml = 2;
```

Comment:
See also here. To ensure backwards compatibility with existing clients `elementsYaml` should have been appended (as field 3).

Reply:
@onematchfox Thanks! I opened this PR https://github.com/argoproj/argo-cd/pull/13335  to address it.